tests/platform_helper/test_command_copilot.py

```diff
@@ -1050,6 +1050,94 @@ def test_appconfig_ip_filter_policy_is_applied_to_each_service_by_default(self,
 
         assert result.exit_code == 0
 
+    @patch("dbt_platform_helper.jinja2_tags.version", new=Mock(return_value="v0.1-TEST"))
+    @patch(
+        "dbt_platform_helper.utils.versioning.running_as_installed_package",
+        new=Mock(return_value=False),
+    )
+    @patch(
+        "dbt_platform_helper.commands.copilot.get_log_destination_arn",
+        new=Mock(
+            return_value='{"prod": "arn:cwl_log_destination_prod", "dev": "arn:dev_cwl_log_destination"}'
+        ),
+    )
+    @patch("dbt_platform_helper.utils.aws.get_aws_session_or_abort", new=Mock())
+    @mock_aws
+    def test_alb_validation_additional_address_list(self, fakefs):
+        """ALB validation should allow additional address list (optional)"""
+        fakefs.create_file(
+            EXTENSION_CONFIG_FILENAME,
+            contents="""
+    alb:
+      type: alb
+      environments:
+        development:
+          cdn_domains_list: 
+            test.domain.uktrade.digital: "test.domain.uktrade.digital"
```

Comment:
I think this would normally be like this...
```suggestion
            test.domain.uktrade.digital: "domain.uktrade.digital"
```